css/style.css

```diff
@@ -781,7 +781,11 @@ input[type="email"] {
   background: transparent;
   width: 100%;
   padding: 15px 20px;
-  font-size: 18px;
+  font: 18px "Manrope-Bold";
+}
+.letter-box .mail-box 
```

Comment:
Selector declared without any CSS properties or separated without the comma. Remove unused selector. Please make sure you read all review comments properly and make changes accordion to them.